.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,70 @@
+---
+name: Release
+about: Cut a Crossplane release
+labels: release
+---
+
+<!--
+Issue title should be in the following format:
+
+    Cut vX.Y.Z Release
+-->
+
+<!--
+Please assign the release owner to the issue.
+-->
+
+### Release Information
+
+**Type:**
+<!-- Select One -->
+- [ ] Minor
+- [ ] Patch
+
+**Version:** <!-- e.g. v1.3.0 -->
+
+**Release Date:** <!-- e.g. Jun 29, 2021 -->
+
+### Steps
+
+<!-- Please complete the following steps in order. -->
+
+I have:
+<!-- Please uncomment the following section only if cutting a minor release. These should be completed at the beginning of [Code Freeze](https://crossplane.io/docs/v1.2/reference/release-cycle.html#code-freeze). -->
+<!--
+- [ ] Created the [release branch](Link to Release Branch, e.g. https://github.com/crossplane/crossplane/tree/release-1.3).
+- [ ] Created and merged an empty commit to the `master` branch ([PR](Link to PR, e.g. https://github.com/crossplane/crossplane/pull/2395)).
+- [ ] Run the [Tag workflow](Link to Tag workflow run) on the `master` branch
+  with the next release candidate tag ([Tag](Link to tag, e.g. v1.4.0-rc.0)).
+-->
+- [ ] Updated all version information in the documentation on the relevant
+  release branch ([PR](<!-- Link to merged PR, e.g. https://github.com/crossplane/crossplane/pull/2386 -->)).
```

Comment:
I think this template would be a lot more readable if we used footer style links here - this would let us keep all the links we needed to insert down in one list at the bottom.

Reply:
@negz do you mean for these links that need to be filled in by the person cutting the release? I fell like that might be somewhat cumbersome to have to match the footer link with the link in the body.

Reply:
Yes, I do mean for these links. This is probably subjective, but do I think using footer links would:

* Make the template easier to read and follow in its raw markdown form (which is how it will mostly be interacted with).
* Make it somewhat easier to insert links (rather than having to inject them into braces inline in the template).

You can use descriptive link names e.g. `[PR][link-to-pr]` to make it very clear what links folks need to fill out down the bottom.

Reply:
My primary concern is less experienced folks not filling out the links because they are not as obvious that they are blank. That being said, I have added a note in a comment and am willing to try this strategy.